kmath-optimization/src/commonMain/tmp/minuit/MnUserTransformation.kt

```diff
@@ -112,9 +112,11 @@ class MnUserTransformation {
         val parm: MinuitParameter = theParameters[theExtOfInt[i]]
         if (parm.hasLimits()) {
             dd = if (parm.hasUpperLimit() && parm.hasLowerLimit()) {
-                theDoubleLimTrafo.dInt2Ext(`val`,
+                theDoubleLimTrafo.dInt2Ext(
+                    `val`,
```

Comment:
Don't you want to rename <code>&#96;val&#96;</code> to `value`? The last has the same name as the first, but also is more clear.